### PR TITLE
Add default stripe minimum

### PR DESCRIPTION
**Pull request by Firstyear**
Due to ongoing issues with download speed, a minimum chunk/stripe download size is required to prevent conflict with TCP slowstart/congestion control.

I'm not 100% sure the patch is correct, the logic in this section is very convoluted and hard to follow. The idea is that if the calculated default blocksize is too small then we set it to a larger value. This way smaller files download in a single shot, but larger ones are chunked over multiple mirrors with a chunk size reasonable enough to actually get a TCP connection to it's full speed.

### CODE REVIEW

**Comment by Firstyear** [comment]
> We already have a minimum chunk size but its currently 4K , as you can see in the `multifetchrequest::makeBlksize` function. We could just change the K to M there to get the same result

I couldn't find it sorry 🤣

But yes I'll adjust this. 4k is FAR too small for a minimum chunk size.

**Comment by Firstyear** [comment]
@bzeller You'll need to squash-merge for this, I can't do a rebase because of #530 :(

**Comment by mlandres** [comment]
Final tests for libzypp-17.32 are running. Probably next week.

**Comment by Firstyear** [comment]
Sounds good, thanks!
